### PR TITLE
[4.0] Fix installation folder not being deleted

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue #32516.

### Summary of Changes
This small PR fixes installation folder not being deleted. See https://github.com/joomla/joomla-cms/issues/32516 for detailed description of the issue

### Testing Instructions
1. Look at https://github.com/joomla/joomla-cms/issues/32516, understand the issue
2. Apply patch, confirm the issue is fixed.

### CODE REVIEW

File: installation/template/js/remove.js [hunk 1]
```diff
@@ -98,6 +98,7 @@ if (document.getElementById('defaultLanguagesButton')) {
           if (successresponse.messages) {
             Joomla.renderMessages(successresponse.messages, '#system-message-container');
           }
+		  Joomla.loadOptions({'csrf.token': successresponse.token});
```
Comment (richard67) [inline installation/template/js/remove.js]:
```suggestion
          Joomla.loadOptions({'csrf.token': successresponse.token});
```
Same indentation as previous line please.

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on e1db87f57f18e53436d0e73d2f2550f65f10c029<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32599">issues.joomla.org/tracker/joomla-cms/32599</a>.</sub>

**Comment by richard67** [comment]
Last code change was just code style, so the previous test result is still valid. I've restored it in the issue tracker so it's correctly counted.

**Comment by infograf768** [comment]
Will test asap.
Shall i make a separate PR for https://github.com/joomla/joomla-cms/issues/32516#issuecomment-788739884 ?

**Comment by joomdonation** [comment]
> Will test asap.
> Shall i make a separate PR for [#32516 (comment)](https://github.com/joomla/joomla-cms/issues/32516#issuecomment-788739884) ?

Yes, please !

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 868a698ab5b11f99cc46ef821e618410acd8573e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32599">issues.joomla.org/tracker/joomla-cms/32599</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32599">issues.joomla.org/tracker/joomla-cms/32599</a>.</sub>

**Comment by infograf768** [comment]
Tks!

**Comment by infograf768** [comment]
Folks, please test https://github.com/joomla/joomla-cms/pull/32601

**Pull request merged**